### ISSUE TITLE
Accepting ip6 addresses, deploying as ip4 fails.

### ISSUE THREAD

**Issue by hcoin** [canonical/microceph#293]
microceph cluster config set cluster_network <ip6address>/<prefix reports no error, treats them as ip4.  As a result, nothing works.  That command should also update ceph.conf

reef/stable

**Comment by UtkarshBhatthere** [comment]
Thanks for the issue @hcoin. I am afraid the description lacks actionable information. Can you please follow the [ISSUE TEMPLATE](https://github.com/canonical/microceph/blob/main/.github/ISSUE_TEMPLATE.md) and populate required information ?

The cluster config CLI does not need to update the ceph.conf file because it uses `ceph config` CLI to internally populate ceph-mon configs. You can check the populated ceph configs using `$ ceph config dump global`.

**Comment by hcoin** [comment]
Just to confirm, is ip6 supported?
Because I experienced such as
#microceph cluster config set cluster_network fc00:1000:0:b00::/64
(or maybe it was  [fc00:1000:0:b00::]/64)

which was accepted, but it broke the cluster.

**Comment by UtkarshBhatthere** [comment]
Yes, there should be no problems configuring IPv6. Cluster Networks are pretty configurable, so you may reset `# microceph cluster config reset cluster_network` and things should go back to normal. In order to find out what went bad, we would need more details.
